src/nodes/sensors.html

```diff
@@ -163,10 +214,18 @@ <h3>Output</h3>
             <label for="node-input-co2"><i class="fa fa-industry"></i> CO2</label>
             <input type="checkbox" id="node-input-co2" />
         </div>
-
+        <div class="form-row">
+            <label for="node-input-separateOutputs"><i class="fa fa-arrows-alt"></i> Separate outputs</label>
+            <input type="checkbox" id="node-input-separateOutputs" />
+        </div>
     </div>
 </script>
 
 <script type="text/html" data-help-name="climate-sensor">
-    <p>Node to trigger scenes</p>
```

Comment:
I started to add some docs there. But then I realized that I don't know the outputs and I don't have a sensor to test it, nor the time now. Could you maybe document those quickly?

Reply:
Added more description :) I completely forgot about the data-help ^^